hedyorg/hedy
# 💻 Add adventure within customize class

Fixes #4084 

**How to test:**
1. Go to `/customize-class`
2. Next to the Reset button (top right), click 'Create adventure'. 
3. Check that the Adventure is correctly added to the level and the class you were in.


https://github.com/hedyorg/hedy/assets/48122190/0a5cb4ee-d231-4328-99fa-d9a86e9b8072
